docs/understanding-tracking-design/create-tracking-plan-with-tracking-scenarios/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Creating a Tracking Plan with Tracking Scenarios"
+sidebar_label: "🆕 Creating a Tracking Plan"
+sidebar_position: 100
+sidebar_custom_props:
+  offerings:
+    - enterprise
+---
+
+As explained in [Introduction to Tracking Design](/docs/understanding-tracking-design/introduction-to-tracking-design/index.md), to use Snowplow successfully, you need to have a good idea of:
+- The events that matter to your business
+- The events that occur in your website, mobile application, server-side systems, factories, call centers, dispatch centers, etc.
+- The decisions you make based on those events
+- What you need to know about those events to make those decisions
+
+While our Schema technology and Data Structures UI can help you capture the structure and instructions for validating your data, we are now introducing Tracking Scenarios to help capture additional information, such as:
+
+- The events you want to capture on your product
+- The channels or apps where they are created
+- The actions the events represent
+- The data structure against which this event will be validated
+- The instructions that need to be applied to correctly populate the event's properties
+- The entities that should be captured together with this event
+
+This additional information represents the tracking requirements that are often lost in translation or difficult to keep up to date. With Tracking Scenarios, teams can create a shared understanding of your business' tracking needs.
+
+
+
+## Tracking Scenario Structure
+
+A Tracking Scenario consists of three main parts.
+
+The descriptive part includes assigning:
```

Comment:
Can we do something to visually break up the continuous long block of text? Like an image of the three parts of tracking sceanrios, or maybe at least add headers for each part?

Reply:
Added some headers